### PR TITLE
Korean translations for cookie and privacy banner

### LINKED ISSUE #3286: Add translations for cookie and privacy banner for 4 live radio v1 services
**Is your feature request related to a problem? Please describe.**
We need to add the cookie / privacy banner translations for the 4live radio v1 services 
(awaiting translations for Tigrinya see: https://github.com/bbc/simorgh/issues/3411

**Describe the solution you'd like**
Add the translations to the service configs. Translations available on spreadsheet (pinned in ws-media slack channel or WSMEDIA-30)

- Afaan Oromoo
- Amharic
- Indonesian
- Korean

Check with UX  to ensure the length of the translations are suitable (not too long)

**Testing notes**
[Tester to complete]

Dev insight: Unlikely to need a tester due to simplicity

**Additional context**
WSMEDIA-30

### PR DESCRIPTION
**Pull request by jroebu14**
Partially resolves https://github.com/bbc/simorgh/issues/3286

**Overall change:** Adds Korean translations for the consent and cookie banner

**Code changes:**

- Adds translations for consent and cookie banner to `src/app/lib/config/services/korean.js`

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

**Testing notes**
Visit http://localhost.bbc.com:7080/korean/bbc_korean_radio/liveradio.

Clear cookies to view the consent banner. Click `OK` to view the cookie banner.

Translations can be verified here https://docs.google.com/spreadsheets/d/1IE6dvHfOJhYTIhC0kg8aRGnaj3fEg1NH9jQwo5NuUDQ/edit#gid=0

### CODE REVIEW

File: src/app/lib/config/services/korean.js [hunk 1]
```diff
@@ -79,18 +79,18 @@ const service = {
           },
           international: {
             first:
-              "We've made some important changes to our Privacy and Cookies Policy and we want you to know what this means for you and your data.",
+              '변경된 개인정보와 쿠키 처리방침이 이용자에게 어떤 영향을 미칠지 알려드립니다',
```
Comment (pjlee11) [inline src/app/lib/config/services/korean.js]:
```suggestion
              '변경된 개인정보와 쿠키 처리방침이 이용자에게 어떤 영향을 미칠지 알려드립니다.',
```

Looks like this translations key wants a fullstop

File: src/app/lib/config/services/korean.js [hunk 2]
```diff
@@ -101,16 +101,16 @@ const service = {
               'https://www.bbc.co.uk/usingthebbc/cookies/what-do-i-need-to-know-about-cookies/',
           },
           international: {
-            first: 'We and our partners use technologies, such as ',
-            linkText: 'cookies',
+            first: 'BBC와 파트너사는 ',
+            linkText: '쿠키',
             last:
-              ', and collect browsing data to give you the best online experience and to personalise the content and advertising shown to you. Please let us know if you agree.',
+              '웹브라우징 데이터 수집 같은 기술을 통해 이용자에게 최적의 온라인 경험을 제공하고 개인에 맞춤화된 콘텐츠와 광고를 노출하고자 합니다. 이에 동의하는지 알려주세요',
```
Comment (pjlee11) [inline src/app/lib/config/services/korean.js]:
```suggestion
              '웹브라우징 데이터 수집 같은 기술을 통해 이용자에게 최적의 온라인 경험을 제공하고 개인에 맞춤화된 콘텐츠와 광고를 노출하고자 합니다. 이에 동의하는지 알려주세요.',
```

**Comment by pjlee11** [comment]
2 points about full stops

**Review by pjlee11 — APPROVED** [review]
LGTM 👍

**Review by FK78 — APPROVED** [review]

**Review by jamesbhobbs — APPROVED** [review]

**Comment by jamesbhobbs** [comment]
What needs manual testing though is that these strings are correct and against the right value.

**Comment by pjlee11** [comment]
@jroebu14 for what reasons was `[x] This PR requires manual testing` added to this? 😄 Assuming it's to ensure the spreadsheet translations values match those in the codebase?

**Comment by staylos92** [comment]
I assume we don't translate https://github.com/bbc/simorgh/pull/3416/files#diff-5dbad3920d358cc2c1410e9abbea5706R76 because we only ever want to show the 'international' text for WS sites?

If that's the case then i'm happy for this to be merged 


---
It does look like the above is correct 👍

**Pull request merged**